packages/openssl/packaging

```diff
@@ -0,0 +1,11 @@
+# abort script on any command that exits with a non zero value
+set -e -x
+
+mkdir ${BOSH_INSTALL_TARGET}/openssl
+echo "Extracting openssl"
+tar xzf openssl/openssl-1.0.2j.tar.gz
```

Comment:
@axelfauvel 

> Default behaviour of keepalived is to send it's logs to syslog. So we need to figure out how to trace the event of failover to implement monitoring

What about proposing to include the [syslog_forwarder](https://github.com/cloudfoundry/syslog-release) to send logs to a remote syslog engine ?

Alternatively, the native smtp alerting [smtp_alert](https://linux.die.net/man/5/keepalived.conf) in keep_alived might also be useful